test/e2e/certificates_test.go

```diff
@@ -5,19 +5,19 @@ package e2e
 
 import (
 	"context"
-	"path/filepath"
-	"time"
-
 	corev1 "k8s.io/api/core/v1"
 	networkingv1 "k8s.io/api/networking/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"path/filepath"
+	"time"
```

Comment:
@geliu2016 not sure why the two import lines are moved? I did not care this in last review, but after recently receiving @swghosh 's https://github.com/openshift/cert-manager-operator/pull/168#discussion_r1431238608 , k8s imports, openshift imports and go-lang built-in imports are better to be in separate blocks?

Reply:
updated.